__docs__/wonder-blocks-typography/typography.argtypes.ts

```diff
@@ -3,24 +3,26 @@ export default {
         control: {type: "text"},
         description: "Text to appear with the specified typography styles.",
         table: {type: {summary: "React.Node"}},
-        type: {required: false},
```

Comment:
Removing these addressed TypeScript errors! These aren't needed since these props are not being marked as required already!